src/main/java/com/github/psambit9791/jdsp/transform/DiscreteFourier.java

```diff
@@ -10,8 +10,12 @@
 
 package com.github.psambit9791.jdsp.transform;
 
+import com.github.psambit9791.jdsp.misc.UtilMethods;
 import org.apache.commons.math3.complex.Complex;
 
+import java.util.Arrays;
+import java.util.stream.Stream;
+
 
```

Comment:
Do you think another function: returnMagPhase() is worthwhile adding? Returns a double[][] with the the first dimension being the length and the second being of size 2 and containing (magnitude, phase).

Reply:
+1